index.d.ts

```diff
@@ -244,11 +246,23 @@ declare module '@lightningjs/blits' {
      * Deprecated: use `this.$trigger()` instead
      */
     trigger: (key: string) => void
-
     /**
      * Router instance
      */
     $router: Router
+    /**
+     * Dynamically set the size of a component holder node
+     */
+    $size: (dimensions: {
+      /**
+       * Component width
+       */
+      w: number,
+      /**
+       * Component height
+       */
+      h: number
+    }) => void
```

Comment:
Hey @michielvandergeest,

Also If possible I would like to pass props to the element with only one object.

Just for example;

```
:props: $props, 
```

Example `props` objects
```
 const props1 = {
  fontFamily: 'Arial',
  fontSize: 20,
 }

const props2 = {
   x: 1,
   y: 2,
   w: 100,
   h: 100,
 }
```